.github/workflows/build-astro-nvim.yml

```
@@ -21,9 +26,91 @@
+          name: latest
+          path: ./artifacts
+
+      - name: Load Docker image
+        run: |
+          docker load < ./artifacts/${{ vars.DOCKER_USERNAME }}__${{ env.IMAGE_NAME }}.tar
+
+      - name: Test Go installation
+        run: |
+          echo "Testing Go installation..."
+          docker run --rm ${{ vars.DOCKER_USERNAME }}/${{ env.IMAGE_NAME }}:latest bash -c "
+            go version && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)